scipy/stats/stats.py

```diff
@@ -2735,9 +2737,9 @@ def iqr(x, axis=None, rng=(25, 75), scale='raw', nan_policy='propagate',
     return out
 
 
-def median_absolute_deviation(x, axis=0, center=np.median, scale=1.4826,
+def median_abs_deviation(x, axis=0, center=np.median, scale='raw',
                               nan_policy='propagate'):
```

Comment:
minor: indentation is off here

Reply:
fixed.

Reply:
there were a few other spots with alignment errors (`E128`s) and `E501`s and which I noticded when I ran pycodestyle, so I fixed those too.